components/cpm/Analysis.vue

```diff
@@ -2,50 +2,107 @@
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${cpmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          CPM experiment
+        </p>
+      </div>
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${rcmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          RCM experiment
+        </p>
+      </div>
+      <div
+        class="bg-no-repeat bg-left-top bg-contain w-1/3 h-full"
+        :style="{backgroundImage: `url(${gcmImage})` }"
+      >
+        <p class="p-3 text-lg prose">
+          GCM experiment
+        </p>
+      </div>
+    </div>
   </div>
 </template>
 
 <script>
 export default {
+  props: {
+    domain: {
+      default: 'AL',
+      type: String
+    }
+  },
   data () {
     return {
       selectedVariable: 'pr',
-      selectedResolution: '3KM',
-      selectedCategory: 'FREQ',
-      selectedDate: '2005-05-30',
+      selectedSeason: 'DJF',
+      selectedModel: 'CLMcom-CMCC-CCLM5-0-9',
       variables: {
-        pr: 'Precipitation'
+        pr: 'Precipitation',
+        tas: 'Temperature'
       },
-      resolution: {
-        '3KM': '3KM',
-        '12KM': '12KM'
+      seasons: {
+        DJF: 'DJF',
+        JJA: 'JJA'
       },
-      categories: {
-        FREQ: 'FREQ',
-        INT: 'INT'
+      models: {
+        'CLMcom-CMCC-CCLM5-0-9': 'CLMcom-CMCC-CCLM5-0-9',
+        'CNRM-AROME41t1': 'CNRM-AROME41t1',
+        'COSMO-pompa': 'COSMO-pompa',
+        'GERICS-REMO2015': 'GERICS-REMO2015',
+        'HadREM3-RA-UM10.1': 'HadREM3-RA-UM10.1',
+        'HCLIMcom-HCLIM38-AROME': 'HCLIMcom-HCLIM38-AROME',
+        'ICTP-RegCM4-7-0': 'ICTP-RegCM4-7-0',
+        'KNMI-HCLIM38h1-AROME': 'KNMI-HCLIM38h1-AROME'
       },
-      dates: {
-        '2005-05-30': '2005-05-30',
-        '2005-08-30': '2005-08-30',
-        '2005-11-30': '2005-11-30',
-        '2006-01-01': '2006-01-01'
-      }
+      modelsList: {
+        'CLMcom-CMCC-CCLM5-0-9': { RCM: 'CCLM4-8-17_ICHEC-EC-EARTH', GCM: 'EC-EARTH' },
+        'CNRM-AROME41t1': { RCM: 'ALADIN63_CNRM-CERFACS-CNRM-CM5', GCM: 'CNRM-CM5' },
+        'COSMO-pompa': { RCM: 'CCLM4-8-17_MPI-M-MPI-ESM-LR', GCM: 'MPI-ESM-LR' },
+        'GERICS-REMO2015': { RCM: 'REMO2015_MPI-M-MPI-ESM-LR', GCM: 'MPI-ESM-LR' },
+        'HadREM3-RA-UM10.1': { RCM: 'MOHC-HadGEM3-GC3.1-N512_MOHC-HadGEM2-ES', GCM: 'HadGEM2-ES' },
+        'HCLIMcom-HCLIM38-AROME': { RCM: 'HCLIMcom-HCLIM38-ALADIN_ICHEC-EC-EARTH', GCM: 'EC-EARTH' },
+        'ICTP-RegCM4-7-0': { RCM: 'ICTP-RegCM4-7-0_MOHC-HadGEM2-ES', GCM: 'HadGEM2-ES' },
+        'KNMI-HCLIM38h1-AROME': { RCM: 'KNMI-RACMO23E_KNMI-EC-EARTH', GCM: 'EC-EARTH' }
+      },
+      rcmImage: 'empty.png',
+      gcmImage: 'empty.png'
     }
   },
   computed: {
-    bgImage () {
-      const fallback = '~/static/cpm_analysis/cpm_prec.png'
+    cpmImage () {
```

Comment:
I think you may have misunderstood what computed properties are. `this.rcmImage` and `this.gcmImage` should not be updated in the computed property for `cpmImage`. It also seems the linter warned you about that, and you explicitly silenced it... 

Perhaps an excel-analogy helps to clarify why this is bad. In excel, you can say: `B1 = A1 - 3` and then `B1` is updated as soon as you change `A1`. But now, you're saying `B1 = A1 - 3 oh and please also change C1 and D1 as follows...`. That's not how excel works.

Computed props in vue are just like excel cells. So the better way to do this is by making a separated computed property out of each of `cpmImage`, `rcmImage`, and `gcmImage`.

Reply:
Same thought as above. I thought `rcmImage` and `gcmImage` should change following the cpm image. But now I got...why the linter warns me all the time :sweat_smile:. （Indeed I got three separate computed properties at the beginning...but then I thought that was too redundant..)

Now they are separated computed properties.